model_compression_toolkit/gptq/keras/quantizer/regularization_factory.py

```diff
@@ -35,7 +36,8 @@ def get_regularization(gptq_config: GradientPTQConfig, representative_data_gen:
     if gptq_config.rounding_type == RoundingType.SoftQuantizer:
         # dry run on the representative dataset to count number of batches
         num_batches = 0
-        for _ in representative_data_gen():
+        rd = representative_data_gen()
```

Comment:
why rd = representative_data_gen() and not just put representative_data_gen() in tqdm?

Reply:
this is just a leftover from something that I checked, good catch thanks